dali/python/nvidia/dali/pipeline.py

```diff
@@ -105,13 +107,25 @@ class Pipeline(object):
 `enable_memory_stats`: bool, optional, default = False
     If DALI should print operator output buffer statistics.
     Usefull for `bytes_per_sample_hint` operator parameter.
+`py_workers_num`: int, optional, default = None
+    Specify the number of Python workers that will process ExternalSource callbacks.
+    The pool starts only if there's at least one ExternalSource with ``parallel`` set to True.
+`py_workers_init` : str, default = "fork"
+    Specify how Python workers should start. Supported methods:
+        * ``fork`` - start by forking the process
+        * ``spawn`` - start fresh interpreter process
+    If ``spawn`` method is used, ExternalSource's callback must be picklable.
+    If you use ``fork``, there must be no CUDA context acquired at the moment of starting
+    the workers. For this reason, if you need to build multiple pipelines that use Python workers,
+    you'll need to call ``start_py_workers`` method before calling ``build`` methods of any of the pipelines.
```

Comment:
Just a nitpick, our docs probably need full forms, so `you will`, there's a `there's` above etc.

Reply:
fix'd